easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml

```diff
@@ -0,0 +1,5 @@
+easyconfigs:
+  - EasyBuild-4.9.3.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21412
+        from-commit: https://github.com/easybuilders/easybuild-easyconfigs/pull/21412
```

Comment:
```suggestion
        from-commit: 1cdd81524c974a29825e37bcf8ef3ccc291f5227
```

Reply:
Oops, copy-paste on my laptop is somewhat broken after upgrading to Ubuntu 24.04 :sweat_smile: